.pre-commit-config.yaml

```diff
@@ -38,25 +38,25 @@ repos:
         args: ["--filter-files"]
       - id: eslint
```

Comment:
@jamescrowley can you remove all for entries that use `invoke_pnpm` from this file? I.e. everything from here to the end of the file.

The script is not functional right now; and there will be superceding work from @fhenrich33 to establish tools for automated FE linting/formatting as layed out in #1400 .